runtime-sdk/src/modules/rofl/types.rs

```diff
@@ -16,6 +18,8 @@ pub struct Create {
     pub policy: AppAuthPolicy,
     /// Identifier generation scheme.
     pub scheme: IdentifierScheme,
+    /// Metadata (arbitrary key/value pairs).
+    pub metadata: BTreeMap<String, String>,
```

Comment:
Create doesn't have Secrets since it cannot be encrypted due to "initial fee payments", right?

Maybe mention this in `AppConfig` docstring, since otherwise it might not be totally clear why we have separate metadata and secrets fields.

Reply:
Yes, exactly. The SEK is only generated once the app is registered as it depends on the assigned app ID. Will add a comment.

Reply:
Updated, please take a look.

Reply:
Thanks!